help_test.go

```diff
@@ -574,58 +574,6 @@ func TestShowCommandHelp_CommandAliases(t *testing.T) {
 	}
 }
 
-func TestHelpNameConsistency(t *testing.T) {
-	// Setup some very basic templates based on actual AppHelp, CommandHelp
-	// and SubcommandHelp templates to display the help name
-	// The inconsistency shows up when users use NewApp() as opposed to
-	// using App{...} directly
```

Comment:
@dearchap sorry I was interpreting this comment to mean that since `NewApp` is removed that testing the inconsistency was no longer valid 🤔